### ISSUE TITLE
Search menu breaks heading structure

### ISSUE THREAD

**Issue by admturner** [washingtonstateuniversity/hrs.wsu.edu#132]
**Describe the bug**

The first heading on a page should be an `h1` tag, but on all pages except the homepage the search button breaks this because it's an `h2` element.
